kong/resource_kong_certificate.go

```diff
@@ -19,14 +19,16 @@ func resourceKongCertificate() *schema.Resource {
 
 		Schema: map[string]*schema.Schema{
 			"certificate": &schema.Schema{
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: false,
+				Type:      schema.TypeString,
+				Required:  true,
+				ForceNew:  false,
+				Sensitive: true,
```

Comment:
@caryyu why does the public key need to be sensitve?

Reply:
You're right, I reverted and only the private key should be sensitive